src/transforms/transform-deployment.ts

```diff
@@ -387,6 +387,7 @@ export const transformDeployment = async (
 			environment: {
 				id: deployment_status.environment,
 				displayName: deployment_status.environment,
+				url: deployment_status.environment_url,
```

Comment:
Unfortunately, according to this public doc https://developer.atlassian.com/cloud/jira/software/rest/api-group-deployments/#api-rest-deployments-0-1-bulk-post
The `url` is not supported in the payload, and confirmed with internal team, the environment tab is not clickable at the moment.

And this is new to features to the ui team, so they don't have a roadmap on making it clickable so far, no ETA yet.

If you like, can you please submit a feature request to Jira software?

Thanks.